### ISSUE TITLE
bloatware in the windows executable

### ISSUE THREAD

**Issue by garypaduana** [gsteph/groove-dl#33]
I was definitely not expecting to see 4+ attempts to install bloatware/malware/spyware in the installer.  That's a huge turnoff on an open source project like this.

**Comment by gsteph** [comment]
Power users know better than to install any of it.
Dumb users already have their systems full of such software.
What else would provide incentive for me to continue supporting the project?

If you hate the installer just the run the two scripts.

**Comment by garypaduana** [comment]
I downloaded the scripts and the installer.  I was curious to see how you packaged it for installation.  Isn't the pursuit itself incentive enough?  (Is this not a hobbyist/enthusiast community?)

**Comment by gsteph** [comment]
Well yes you're correct (this is why we're on Github right now commenting on open source code) but sometimes it's such a hassle to have to go through Grooveshark's code all over again to find what they changed. This provides the extra edge.

I packaged it using an NSIS script provided by the guys behind the bloatware ads XD

I'm aware it's a turn off and it was such a hard decision to go through this route.. but you'd be surprised at how many actually care about this. You're the first so far.

Sorry if I caused you any inconvenience.
